src/pages/cartPage/CartPage.jsx

```diff
@@ -1,71 +1,102 @@
+              <div className="dui-cart-details-card">
+                <h3 className="dui-util-fw-bld">PRICE DETAILS</h3>
+                <hr />
+                <div className="dui-util-disp-flex-row">
+                  <p>Price ({getCartItemCount(cart)} items)</p>
+                  <p className="mgn-left-auto">
+                    ${getTotalOriginalPriceOfCart(cart)}
+                  </p>
                 </div>
-
-                <div className="cart-container__price-details">
-
-                    <div className="dui-cart-details-card">
-                        <h3 className="dui-util-fw-bld">PRICE DETAILS</h3>
-                        <hr/>
-                        <div className="dui-util-disp-flex-row">
-                            <p>Price (2 items)</p>
-                            <p className="mgn-left-auto">$4999</p>
-                        </div>
-                        <div className="dui-util-disp-flex-row">
-                            <p>Discount</p>
-                            <p className="mgn-left-auto">-$1999</p>
-                        </div>
-                        <div className="dui-util-disp-flex-row">
-                            <p>Delivery Charges</p>
-                            <p className="mgn-left-auto">$499</p>
-                        </div>
-                        <hr/>
-                        <div className="dui-util-disp-flex-row dui-util-fw-bld">
-                            <p>TOTAL AMOUNT</p>
-                            <p className="mgn-left-auto">$3499</p>
-                        </div>
-                        <hr/>
-                        <p>You will save $1999 on this order</p>
-
-                        <a className="dui-link dui-link--primary dui-util-txt-sm dui-util-spc-pad-0_8rem-xs dui-util-fw-sbld dui-util-txt-align-cent"
-                            href="#">
-                            PLACE ORDER
-                        </a>
-                    </div>
+                <div className="dui-util-disp-flex-row">
+                  <p>Discount</p>
+                  <p className="mgn-left-auto">
+                    -${getTotalDiscountOnCart(cart)}
+                  </p>
+                </div>
+                <div className="dui-util-disp-flex-row">
+                  <p>Delivery Charges</p>
+                  <p className="mgn-left-auto">${getDeliveryCharges(cart)}</p>
                 </div>
+                <hr />
+                <div className="dui-util-disp-flex-row dui-util-fw-bld">
+                  <p>TOTAL AMOUNT</p>
+                  <p className="mgn-left-auto">
+                    $
+                    {Number(getTotalSalePriceOfCart(cart)) +
+                      Number(getDeliveryCharges(cart))}
+                  </p>
+                </div>
+                <hr />
+                <p>
+                  You will save ${getTotalDiscountOnCart(cart)} on this order
+                </p>
+
+                <a
```

Comment:
If possible convert a tag into a Link tag.

Reply:
will make it part of refactoring.